scripts/ta_bin_to_c.py

```diff
@@ -40,8 +40,7 @@ def get_args():
 
 
 def get_name(obj):
-    # Symbol or section .name might be a byte array or a string, we want a
-    # string
+    # Symbol or section .name can be a byte array or a string, we want a string
```

Comment:
I copy pasted get_name() with this description from some other place. Probably you need to make 2 places look the same..

Reply:
True but this commit has no reason to change an unrelated file.